cabal.project

```diff
@@ -22,8 +22,12 @@ packages:
 package cardano-cli
   ghc-options: -Werror
 
-package cardano-api
-  ghc-options: -Werror
+source-repository-package
+  type: git
+  location: https://github.com/IntersectMBO/cardano-api.git
+  tag: f59090668707b8960a057435a507924ae18be16e
+  subdir: cardano-api
+  --sha256: sha256-TJeLlUmZhlckBqQNQVSyFySM66bWa9ggIIOtnNBi0Hs= 
```

Comment:
Since - generally speaking - we don't allow SRPs on `main`,  you'll need to wait for `cardano-api` to be released and updated to merge.